support/util/imagemetadata.go

```diff
@@ -183,7 +182,10 @@ func (r *RegistryClientImageMetadataProvider) GetDigest(ctx context.Context, ima
 	case len(composedParsedRef.Tag) > 0:
 		desc, err := repo.Tags(ctx).Get(ctx, composedParsedRef.Tag)
 		if err != nil {
-			return "", nil, err
+			fmt.Printf("failed to get repository tags for %s composedParsedRef: %+v: %v. Falling back to the original imageRef %s.\n", composedParsedRef.Tag, composedParsedRef, err, imageRef)
+			if desc, err = fallbackToOriginalImageRef(ctx, imageRef, pullSecret); err != nil {
```

Comment:
Why do we need handling the fallback at all? shouldn't the container runtime handle that as for any icsp?